mmv1/products/compute/OrganizationSecurityPolicyRule.yaml

```diff
@@ -88,12 +88,13 @@ properties:
       - name: 'versionedExpr'
         type: Enum
         description: |
-          Preconfigured versioned expression. For organization security policy rules,
-          the only supported type is "FIREWALL".
+          Preconfigured versioned expression. If this field is specified, config must also be specified.
+          Available preconfigured expressions along with their requirements are: SRC_IPS_V1
+          - must specify the corresponding srcIpRange field in config.
         min_version: 'beta'
-        default_value: "FIREWALL"
+        default_value: 'SRC_IPS_V1'
```

Comment:
let's switch to `default_from_api: true` instead of switching the default, which is a breaking change. 
We should probably support both enum values for legacy